### PR TITLE
Added the documentation for Runtime View

### PR DESCRIPTION
**Pull request by albertodb99**

### CODE REVIEW

File: webapp/docs/06_runtime_view.adoc [hunk 1]
```diff
@@ -29,29 +29,87 @@ There are many notations for describing scenarios, e.g.
...
+=== Alice wants to know if Bob is near here, and Bob allows Alice to see him in the radar
 
 
-* _<insert runtime diagram or textual description of the scenario>_
-* _<insert description of the notable aspects of the interactions between the
-building block instances depicted in this diagram.>_
+* _Today, Alice is bored at home, but she wants to do something, taking into account COVID-19 restrictions.
+    As a consequence of these restrictions, she cannot leave her town, and wants to know if his best friend Bob
+    who lives in the same town as her, is near. As Bob allows Alice to find her in the application, Alice is able to access Bob's location in case he is near her._
+* _As you can see in the diagram below, at first Alice has to authenticate and allow sharing his location in order to find his mutuals between a proximity radius.
```
Comment (UO266499) [inline webapp/docs/06_runtime_view.adoc:40]: and allow sharing her location

his -> her 
as we are talking about Alice not Bob

File: webapp/docs/06_runtime_view.adoc [hunk 2]
```diff
@@ -29,29 +29,87 @@ There are many notations for describing scenarios, e.g.
...
+App  -> Pod2: Asks for Bob's permission to share his location
+Pod2 -> Bob: Sends a petition
+Pod2 <-- Bob: Accepts to share his location
+App <-- Pod2: Sends the response
+Pod1 <-- App: Manages the response, and shows Bob in the radar
+Alice <-- Pod1: Radar with Bob's location
+----
+=== Alice wants to know if Bob is near here, and Bob doesn't want Alice to see him in the radar
+* _For this case, we have the same exact context as in the previous use case, but this time, Bob is at home
+    watching some sitcoms, and is not allowing nothing to disturb him, so it refuses to connect with Alice in the App_
```
Comment (UO266499) [inline webapp/docs/06_runtime_view.adoc]: is not allowing anything to

File: webapp/docs/06_runtime_view.adoc [hunk 3]
```diff
@@ -29,29 +29,87 @@ There are many notations for describing scenarios, e.g.
...
+* _As you can see in the diagram below, at first Alice has to authenticate and allow sharing his location in order to find his mutuals between a proximity radius.
```
Comment (UO266499) [inline webapp/docs/06_runtime_view.adoc]: allow to share her location

**Comment by ghost** [comment]
I have reviewed the changes commented and commited by Alex, and I think that the .adoc is correct then

**Pull request merged**